pkg/utils/utils.go

```diff
@@ -141,3 +141,31 @@ func GetClusterID(cd *hivev1.ClusterDeployment) string {
 		return cd.Spec.ClusterName
 	}
 }
+
+// IsRedHatHyperShiftInfrastructure returns whether or not a cluster is part of the Red Hat HyperShift infrastructure
+func IsRedHatHyperShiftInfrastructure(cd *hivev1.ClusterDeployment) bool {
```

Comment:
Function name isn't consistent with configuration.  Recommend:

```suggestion
func IsRedHatInfrastructure(cd *hivev1.ClusterDeployment) bool {
```